cmd/root.go

```diff
@@ -382,6 +391,23 @@ func AskForConfirmation(s string) (res bool, err error) {
 	}
 }
 
+func EventsMatch(a, b []string) bool {
```

Comment:
Had to create custom comparison function for slices. No built-in method for this.

Reply:
That should be fine.